microsite/data/plugins/blackduck.yaml

```diff
@@ -0,0 +1,12 @@
+---
+title: BlackDuck
+author: Deepankumar
+authorUrl: https://github.com/deepan10
+category: Security
+description: BlackDuck scanned vulnerabilities in Backstage.
+documentation: https://github.com/deepan10/backstage-plugin-blackduck
+iconUrl: img/blackduck-icon.svg
```

Comment:
Do you have permission to use this logo?

Reply:
Hello,
I took the logo from VectorLogo - https://www.vectorlogo.zone/logos/blackducksoftware/index.html and BlackDuck is an opensource software.

Reply:
Neither of those mean we have permission to use the logo unfortunately. I can't find many usages of the logo other than in social media avatars though, so I'm not quite for what's up with the state of it. We'll need permission to use the logo directly, or work around it by using a custom logo.

Reply:
I update the `iconUrl` with the URL directly from GitHub, like how the other plugins are used.

Reply:
Ah right, that's another option :grin: thanks!